src/core/api_client/tests/_clientTestsContext.ts

```diff
@@ -1,6 +1,7 @@
 import MockAdapter from 'axios-mock-adapter';
 import { base64UrlDecode, base64UrlEncode } from './_tokenUtils';
 import './_configMock';
+// eslint-disable-next-line import/order
 import { ApiClient } from '../apiClient';
```

Comment:
just put `import { ApiClient } from '../apiClient';` in line 2, and no need in `// eslint-disable-next-line import/order`